requirements.txt

```diff
@@ -2,4 +2,6 @@
 black==22.3.0
 glom==20.11.0
 pytest==6.2.5
+mypy==0.990
+msgpack-types==0.2.0
```

Comment:
Is this being explicitly used somewhere?

Reply:
@algochoi https://github.com/sbdchd/msgpack-types supplies type stubs.  Without it, I see the following errors:

```
algosdk/future/transaction.py:4: error: Skipping analyzing "msgpack": module is installed, but missing library stubs or py.typed marker  [import]
algosdk/future/transaction.py:4: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

Reply:
Ah thank you. I was a worried at using a 0.x versioned package (might become stale or have breaking changes), but that seems okay to me.